javascripts/domManager.js

```diff
@@ -0,0 +1,122 @@
+//*****DOM*****//
+/*global battledome: true */ 
+'use strict';
+
+var battledome = (function(battledome){
+
+	//ADDS GAME SETUP VIEW TO DOM//
+	battledome.addGameSetupView = () => {
+		$('.mainContent').html(`
+			<div class='row'>
+				<div class='col-sm-3 col-sm-offset-2 row'>
+					<input class="nameInput col-sm-12" type="text" placeholder="player A name:">
+					<select class="playerModel col-sm-12">
+						<option value="Quadcopter">Quadcopter</option>
+						<option value="Hovership">Hovership</option>
+						<option value="LaserMech">LaserMech</option>
+						<option value="GunMech">GunMech</option>
+						<option value="ArmorBuggy">ArmorBuggy</option>
+						<option value="FlameTank">FlameTank</option>
+					</select>
+				</div>
+				<div class='col-sm-3 col-sm-offset-2 row'>
+					<input class="nameInput col-sm-12" type="text" placeholder="player B name:">
+					<select class="playerModel col-sm-12">
+						<option value="Quadcopter">Quadcopter</option>
+						<option value="Hovership">Hovership</option>
+						<option value="LaserMech">LaserMech</option>
+						<option value="GunMech">GunMech</option>
+						<option value="ArmorBuggy">ArmorBuggy</option>
+						<option value="FlameTank">FlameTank</option>
+					</select>
+				</div>	
+			</div>
+			<div class='row'>
+				<button class='col-sm-4 col-sm-offset-4 btn btn-primary' id="initializeCombatButton" type="submit">Submit</button>
```

Comment:
Since you're not submitting a form with the button, it doesn't need the `type="submit"` attribute.